### ISSUE TITLE
v3.3.2 Installation errors on database 

### ISSUE THREAD

**Issue by prakash991** [iomad/iomad#750]
Hi, 
I am getting the following errors with respect to the mysql db during installation of iomad v3.3.2. I don't have superuser privileges and i tried to command line and phpmyadmin options to set these db settings through cpanel. It is not working. Is it possible to correct these errors through some other means? Thx in advance for your help. 

ERRORS:
mysql_full_unicode_support#File_format | this test must pass
Your database has tables using Antelope as the file format. Full UTF-8 support in MySQL and MariaDB requires the Barracuda file format. Please convert the tables to the Barracuda file format. See the documentation Administration via command line for details of a tool for converting InnoDB tables to Barracuda. |   

mysql_full_unicode_support#Large_prefix | this test must passFor full support of UTF-8 both MySQL and MariaDB require you to change your MySQL setting 'innodb_large_prefix' to 'ON'. See the documentation for further details. |  

site not https | if this test fails, it indicates a potential problemIt has been detected that your site is not secured using HTTPS. It is strongly recommended to migrate your site to HTTPS for increased security and improved integration with other systems.

**Comment by turf212** [comment]
The only issues you need to address is the character set and collation for your database.  As of Moodle 3.3 this needs to be utf8mb4 and not utf8.

**Comment by prakash991** [comment]
Yes I understand but since our domain is hosted in a shared hosting environment, the hosting provider is asking us to buy a Virtual Private Server to set these global variables. Is there a workaround?

**Comment by turf212** [comment]
Its how the database is created for Moodle rather than the default settings for the database server.  They should be able to run different databases with different character sets and collations.

create database mydatabase default character set utf8mb4 collate utf8mb4_unicode_ci;

Or if you have alter database privs you can change the characterset / collation.

**Comment by prakash991** [comment]
Thanks Derick. I tried creating the database and then changed the collation through phpmyadmin to utf8mb4_unicode_ci. I am not a moodle nor a database expert but here is what my analysis is at this point:

1) A direct installation of Moodle 3.3.2 is working well. How did I install this? Through godaddy (my hosting provider) has applications including Moodle that can be installed through Installatron.
2) If the above is working then iomad should "theoretically" work too
3) There are two errors that iomad installation is pointing to as I had given previously - changing Antelope to Barracuda file format and turning innodb_large_prefix to ON. Both of these seems like have to be changed. I tried the command line sql scripts as recommended without any luck as it says I need root access which I don't have. Moreover, godaddy is saying that I will need to purchase VPS to get root access which costs. 

My main objective of using iomad is to try the multi-tenancy feature which will work very well for us as we work with more than 25 schools across India and we will be signing up more. If you guys can help that will be great. Thanks in advance.
